leases/tests/test_lease_mutations.py

```diff
@@ -0,0 +1,227 @@
+import uuid
+from random import randint
+
+import pytest
+from freezegun import freeze_time
+from graphql_relay import to_global_id
+
+from berth_reservations.tests.utils import (
+    assert_doesnt_exist,
+    assert_field_missing,
+    assert_in_errors,
+    assert_not_enough_permissions,
+    GraphQLTestClient,
+)
+from leases.models import BerthLease
+
+client = GraphQLTestClient()
+
+GRAPHQL_URL = "/graphql_v2/"
+
+CREATE_BERTH_LEASE_MUTATION = """
+mutation CreateBerthLease($input: CreateBerthLeaseMutationInput!) {
+    createBerthLease(input:$input){
```

Comment:
IIRC in other tests indent was 2 spaces 😅  also, here some of the nested indents are 2 spaces down below.

Reply:
There is a mixture. Some of the older tests have 4 spaces, some others have 2. I'll normalise all to 4 spaces (#python) and create a separate PR to avoid cluttering this one.

Reply:
yes, good 👍